extensions/positron-notebook-controllers/src/test/notebookController.test.ts

```diff
@@ -0,0 +1,200 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2023-2024 Posit Software, PBC. All rights reserved.
```

Comment:
```suggestion
 *  Copyright (C) 2024 Posit Software, PBC. All rights reserved.
```
Tiny nit! Don't think it matters